File: Python/Programmars/Level_1/약수의 합.py
def solution(n):
    answer = 0
    for i in range(1, n+1):
        if n % i == 0:
            answer += n//i
    return answer


print(solution(12))  # 28
print(solution(5))  # 6


# ref: 한 줄로 표현
def solution(num):
    return sum([i for i in range(1, num+1) if num % i == 0])

File: JavaScript/Programmars/Level_1/약수의 합.js
function solution(n) {
  let answer = 0;
  for (let i = 1; i <= n; i++) {
    if (n % i === 0) answer += i;
  }
  return answer;
}

console.log(solution(12)); // 28
console.log(solution(5)); // 6
